Packs/ExabeamDataLake/Integrations/ExabeamDataLake/ExabeamDataLake_description.md

```diff
@@ -0,0 +1,9 @@
+Exabeam Data Lake provides a highly scalable, cost-effective, and searchable log management system. Data Lake is used for log collection, storage, processing, and presentation.
+
+### Authentication Method
+ - **Basic Authentication** - Providing Username and password in the corresponding   parameters in the configuration.
```

Comment:
format the text please.
Do we have some further info on how to obtain the password and username?